media-library/overview/folders.md

```diff
@@ -22,3 +22,13 @@ To add files to a particular folder, you need to open the folder first.
 2. You can now drag and drop images, or use the popup-based file selection to upload files to the selected folder.
 
 In case you have any questions regarding adding files to the folder, please get in touch with our team at [support@imagekit.io](mailto:customer-support@imagekit.io).
+
+### Asset ownership
+
+If a new folder is created through the ImageKit's media library, the creator of the folder will be the "owner" of the folder. Asset ownership governs the access and permission level of the user on an asset. They would have "manage" access to the folder and all its subfolders and files. Learn how asset ownership controls access and permission levels [here](../../access-control-and-permissions/README.md#access-and-permission-management).
+
+The folder will not have any owner if it is not created via the dashboard. Note that when creating a new folder at a location where a folder with the same name already exists, the owner of the folder does not change.
```

Comment:
In case of no ownership info, the asset still has an owner - the account. We should specify this, possibly like this:

The folder will not have any ownership info if it is not created via the dashboard, e.g. via the API or an SDK.

Reply:
done